LondonBritishLibrary/orient/BLorient540.xml

```diff
@@ -0,0 +1,220 @@
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <msDesc xml:id="ms">
+                    <msIdentifier>
+                        <repository ref="INS00001BL"/>
+                        <collection>Oriental</collection>
+                        <idno>BLorient 540</idno>
+                        <altIdentifier>
+                            <idno>Wright cat. CXX</idno>
+                        </altIdentifier>
+                        <altIdentifier>
+                            <idno>Wright 120</idno>
+                        </altIdentifier>
+                    </msIdentifier>
+                    
+                    <msContents>
+                        <summary/>
+                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="3r"/>
+                            <title type="complete" ref="LIT2701Dawit"/>
+                            <textLang mainLang="gez"/>
+                            
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="3r"/>
+                                <title type="complete" ref="LIT3985Mazmura"/>
+                                
+                                <msItem xml:id="ms_i1.1.1">
+                                    <locus from="3r"/>
+                                    <title type="complete" ref="LIT3985Mazmura#IntroD"/>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.2">
+                                    <title type="complete" ref="LIT3985Mazmura#Psalms"/>
+                              </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.3">
+                                    <locus from="149r"/>
+                                    <title type="complete" ref="LIT3985Mazmura#Canticles"/>
+                                </msItem>
+                                
+                                <msItem xml:id="ms_i1.1.4">
+                                    <locus from="164r"/>
+                                    <title type="complete" ref="LIT3985Mazmura#SongOfSongs"/>
+                                </msItem>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="171v"/>
+                                <title type="complete" ref="LIT2509Weddas"/>
+                            </msItem>
+                            
```

Comment:
Is there a reason for a gap?

Reply:
Do you mean in the msItem numeration? No, that was a mistake